server/src/main/java/org/elasticsearch/search/aggregations/support/CoreValuesSourceType.java

```diff
@@ -224,6 +224,72 @@ public ValuesSource getField(FieldContext fieldContext, AggregationScript.LeafFa
         public ValuesSource replaceMissing(ValuesSource valuesSource, Object rawMissing, DocValueFormat docValueFormat, LongSupplier now) {
             throw new IllegalArgumentException("Can't apply missing values on a " + valuesSource.getClass());
         }
+    },
+    // TODO: Ordinal Numbering sync with types from master
+    IP{
```

Comment:
I'm not sure how I feel about pulling the specialized values source types here into Master without the code that uses them.  That seems potentially confusing without context - e.g. We have an IP type now, but VSConfig will still resolve IP fields to Bytes because this isn't wired up.  At a minimum, we should leave a comment in big letters that this is test only until the refactor lands (which hopefully is soon, but still)